spec/models/test_track/ab_configuration_spec.rb

```diff
@@ -68,7 +68,7 @@
     it "raises a descriptive error when the split is not in the split_registry" do
       expect do
         described_class.new initialize_options.merge(split_name: :not_a_real_split)
-      end.to raise_error("unknown split: not_a_real_split")
+      end.to raise_error("unknown split: not_a_real_split. You may need to run rake test_track:schema:load")
```

Comment:
You kind of need to get into env overriding if you want to actually test this conditional display feature too, which I think is worth it because we don't want it to start popping up in production environments later.